src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/UpdateInstanceNewWatermark.sql

```diff
@@ -0,0 +1,53 @@
+﻿/*************************************************************
+    Stored procedures for updating an instance status.
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     UpdateInstanceNewWatermark
+--
+-- DESCRIPTION
+--     Updates a DICOM instance NewWatermark
+--
+-- PARAMETERS
+--     @partitionKey
+--         * The system identified of the data partition.
+--     @studyInstanceUid
+--         * The study instance UID.
+--     @seriesInstanceUid
+--         * The series instance UID.
+--     @sopInstanceUid
+--         * The SOP instance UID.
+--
+-- RETURN VALUE
+--     None
+--
+CREATE OR ALTER PROCEDURE dbo.UpdateInstanceNewWatermark
```

Comment:
Maybe you want to do this batch? Again the Job WF will help you here

Reply:
Updated to be batch update